Java/src/main/java/com/gildedrose/item/CustomisedItemFactory.java

```diff
@@ -0,0 +1,23 @@
+package com.gildedrose.item;
+
+public class CustomisedItemFactory {
+
+    public final static String SULFURAS = "Sulfuras, Hand of Ragnaros";
```

Comment:
For info only, By default intelliJ marks constants as public, however it is worth changing them to something more restrictive in cases where they are only used within one class.